README.md

```diff
@@ -103,6 +104,17 @@ Starting to look for vaccine slots for Roger Philibert in 5 next day(s)...
 This may take a few minutes/hours, be patient!
 ```
 
+### Look on specific date
+
+By default, the script looks for slots between now and next day at 23:59:59. If you can't be vaccinated right now (e.g covid in the last 3 months or out of town) and you are looking for an appointment in a distant future, you can pass a starting date:
+
+```
+$ ./doctoshotgun.py paris roger.philibert@gmail.com --start-date 17/06/2021
+Password:
+Starting to look for vaccine slots for Roger Philibert in 5 next day(s)...
```

Comment:
By default it is 5 days I guess, and perhaps display in output from which date it starts?

Reply:
I modified the log